roles/matrix-base/templates/static-files/well-known/matrix-client.j2

```diff
@@ -26,12 +26,13 @@
 		"preferredDomain": {{ matrix_client_element_jitsi_preferredDomain|to_json }}
 	}
 	{% endif %}
-	{% if not matrix_client_element_e2ee_default %},
+	,
 	"io.element.e2ee": {
-		"default": false
+		"default": {{ matrix_client_element_e2ee_default|to_json }},
+		"secure_backup_required": {{ matrix_client_element_e2ee_backup_required|to_json }},
```

Comment:
Oh, sorry, it's just now that I noticed this!

Would you rename these variables to be more in line with the actual options that they control?

- `matrix_client_element_e2ee_backup_required` -> `matrix_client_element_e2ee_secure_backup_required` (because of `secure_backup_required`)
- `matrix_client_element_e2ee_backup_methods` -> `matrix_client_element_e2ee_secure_backup_setup_methods` (because of `secure_backup_setup_methods`)

We try to be consistent like this everywhere in the playbook.

Reply:
sure, done. Thanks for the review.